hmda/src/main/scala/hmda/persistence/submission/HmdaProcessingUtils.scala

```diff
@@ -53,26 +53,31 @@ object HmdaProcessingUtils {
     }
   }
 
-  def updateSubmissionReceipt(sharding: ClusterSharding, submissionId: SubmissionId, timestamp: Long, receipt: String, log: Logger)(
+  def updateSubmissionStatusAndReceipt(sharding: ClusterSharding, submissionId: SubmissionId, timestamp: Long, receipt: String, modified: SubmissionStatus, log: Logger)(
     implicit ec: ExecutionContext,
     timeout: Timeout
   ): Unit = {
     val submissionPersistence: EntityRef[SubmissionCommand] =
       sharding.entityRefFor(SubmissionPersistence.typeKey, s"${SubmissionPersistence.name}-$submissionId")
 
+    val submissionManager =
+      sharding.entityRefFor(SubmissionManager.typeKey, s"${SubmissionManager.name}-$submissionId")
+
     val fSubmission: Future[Option[Submission]] = submissionPersistence ? (ref => GetSubmission(ref))
 
     for {
-      s <- fSubmission
-      m = s
+      m <- fSubmission
```

Comment:
Since we're making changes here can we update to better variable names?

Reply:
Good point